en_us.lang

```diff
@@ -280,81 +280,81 @@ gui.xaero_24h=24h
+gui.xaero_box_cave_maps=Mapping mode that can go deeper than the surface blocks, mainly to display underground caves and interiors of buildings. The ceiling size refers to the size of a horizontal "square" of solid blocks that has to be detected above you to activate the cave mode.
 gui.xaero_box_entity_amount=The limit per each entity category.
-gui.xaero_box_distance=Which ingame waypoints to show the distance to for.
-gui.xaero_box_always_distance=Display distance even when the waypoint is very close. This does not override the "Distance To WP" option.
-gui.xaero_box_arrow_scale=Scale for the arrow used in the non-rotating variant of the minimap and some other cases.
-gui.xaero_box_arrow_color=Color of the arrow used in the non-rotating variant of the minimap and some other cases.
-gui.xaero_box_lock_player_heads=Display player heads instead of coloured dots even if the TAB key is not pressed.
-gui.xaero_box_cave_zoom=How much to zoom further in when the cave maps mode is active.
-gui.xaero_box_entity_depth=Display entities darker depending on their Y level relative to you.
-gui.xaero_box_height_limit=Maximum Y level distance for entities until they are no longer displayed.
+gui.xaero_box_distance=Specifies, for which in-game waypoints to show the distance to.
+gui.xaero_box_always_distance=Display distance even when the waypoint is very close. This setting does not override the "Distance to WP" option.
+gui.xaero_box_arrow_scale=Scale of the arrow used in the non-rotating variant of the minimap and several other cases.
+gui.xaero_box_arrow_color=Color of the arrow used in the non-rotating variant of the minimap and several other cases.
+gui.xaero_box_lock_player_heads=Display player heads instead of colored dots even if the Tab key is not pressed.
+gui.xaero_box_cave_zoom=Specifies, how much to zoom the minimap further in when the cave mode is active.
+gui.xaero_box_entity_depth=Display entities darker on the minimap depending on their Y level relative to you.
+gui.xaero_box_height_limit=Maximum Y-level distance for entities until they are no longer displayed on the minimap.
 gui.xaero_box_compass_over_wp=Render compass letters (N, E, S, W) over the on-map waypoints.
-gui.xaero_toggle_map_waypoints=Toggle On-map Waypoints
+gui.xaero_toggle_map_waypoints=Toggle On-Map Waypoints
 gui.xaero_always_entity_nametags=Always Render Entity Name Tags
 gui.xaero_minimap_text_align=Info Text Alignment
 gui.xaero_center=Center
 gui.xaero_left=Left
 gui.xaero_right=Right
-gui.xaero_show_angles=Show View Angles
+gui.xaero_show_angles=Display Facing Angles
 gui.xaero_box_always_entity_nametags=Always render names for entities with name tags regardless of what the name setting is set to in the "Entity Radar" screen.
```

Comment:
For consistency reasons we should choose only one of them and use it in all places:
* on the screen
* in the screen

Do you know which one is correct? I always thought that only the 1st one was correct, but I may have been wrong.